rocky/katalogus/views/boefje_setup.py

```diff
@@ -85,45 +62,123 @@ class AddBoefjeVariantView(BoefjeSetupView):
     def setup(self, request, *args, **kwargs):
         super().setup(request, *args, **kwargs)
 
-        plugin_id = self.kwargs["plugin_id"]
+        self.plugin_id = str(uuid.uuid4())
```

Comment:
Personally I think it's cleaner to leave `plugin_id` and `created` types as is, and only cast it to a `str` type whenever it's actually needed